ibis/pandas/udf.py

```diff
@@ -100,6 +101,29 @@ def arguments_from_signature(signature, *args, **kwargs):
     return args, new_kwargs
 
 
+def create_gens_from_args_groupby(args):
+    """ Create generators for each args for groupby udaf.
+
+    If the arg is SeriesGroupBy, return a generator that outputs each group.
+    If the arg is not SeriesGroupBy, return a generator that repeats the arg.
```

Comment:
what else could this be? can you type it

Reply:
Improved docstring